dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerValidationReport.java

```diff
@@ -57,6 +58,11 @@
     @JsonProperty
     @Builder.Default
     private List<TrackerValidationHookTimerReport> performanceReport = new ArrayList<>();
+
+    @JsonProperty
+    @Builder.Default
+    private List<TrackerDto> notValidDtos = new ArrayList<>();
```

Comment:
is "invalidDtos" a better name? or maybe there is a reason for the naming. Just curious.

Reply:
It was probably validDtos and then I added the not.
InvalidDtos makes much more sense :)